MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -33,6 +35,10 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
 	
+	func display(_ viewModel: FeedErrorViewModel) {
+		errorView.show(message: viewModel.errorMessage)
```

Comment:
Here the view controller needs to check if there's a message or not.

If there's a message, the view controller should call `show()`
If the message is `nil`, the view controller should call `hideMessage()`

```suggestion
		if let message = viewModel.errorMessage {
			errorView.show(message: message)
		} else {
			errorView.hideMessage()
		}
```